target/snitch_cluster/sw/snax/xdma/include/snax-xdma-csr-addr.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 KU Leuven.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Yunhao Deng <yunhao.deng@kuleuven.be>
+
+// This file is (will be) generated by scala, do not modify it manually
+
+#define XDMA_BASE_ADDR 960
+#define XDMA_SRC_ADDR_PTR_LSB XDMA_BASE_ADDR
+#define XDMA_SRC_ADDR_PTR_MSB XDMA_SRC_ADDR_PTR_LSB + 1
+#define XDMA_SRC_DIM 3
+#define XDMA_SRC_BOUND_PTR XDMA_SRC_ADDR_PTR_MSB + 1
+#define XDMA_SRC_STRIDE_PTR XDMA_SRC_BOUND_PTR + XDMA_SRC_DIM
+#define XDMA_SRC_EXT_CSR_PTR XDMA_SRC_STRIDE_PTR + XDMA_SRC_DIM
+#define XDMA_SRC_EXT_NUM 0
+#define XDMA_SRC_EXT_CSR_NUM 0
+#define XDMA_SRC_EXT_CUSTOM_CSR_NUM \
+    {}
+#define XDMA_DST_ADDR_PTR_LSB XDMA_SRC_EXT_CSR_PTR + XDMA_SRC_EXT_CSR_NUM
+#define XDMA_DST_ADDR_PTR_MSB XDMA_DST_ADDR_PTR_LSB + 1
+#define XDMA_DST_DIM 3
+#define XDMA_DST_BOUND_PTR XDMA_DST_ADDR_PTR_MSB + 1
+#define XDMA_DST_STRIDE_PTR XDMA_DST_BOUND_PTR + XDMA_DST_DIM
+#define XDMA_DST_EXT_CSR_PTR XDMA_DST_STRIDE_PTR + XDMA_DST_DIM
+#define XDMA_DST_EXT_NUM 3
+#define XDMA_DST_EXT_CSR_NUM 5
+#define XDMA_DST_EXT_CUSTOM_CSR_NUM \
+    { 1, 1, 0 }
+#define XDMA_START_PTR XDMA_DST_EXT_CSR_PTR + XDMA_DST_EXT_CSR_NUM
+#define XDMA_COMMIT_TASK_PTR XDMA_START_PTR + 1
+#define XDMA_FINISH_TASK_PTR XDMA_COMMIT_TASK_PTR + 1
```

Comment:
Nice!

Reply:
This part will be generated by chisel, but not implemented yet. It will be done in the next PR.